.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: objective-c
+
+# macOS and Xcode Version
+# Xcode 9.2 running on macOS 10.12
+osx_image: xcode9.2
+
+# Build dependencies
+install:
+  - brew install python3
+  - sudo gem install asciidoctor
+  - ( cd External && ./makeAll )
```

Comment:
Will this ensure the submodules are pulled in?

Reply:
Travis automatically checks out the repo with the submodules.